src/Core/MailTemplate/Theme.php

```diff
@@ -47,6 +48,7 @@ class Theme implements ThemeInterface
     public function __construct($name)
     {
         $this->name = $name;
+        $this->layouts = new LayoutCollection();
```

Comment:
any reason to add this here? You can use `setLayouts` instead, or use constructor like 

```php
public function __construct($name, $layouts = null)
{
    $this->layouts = $layouts === null ? new LayoutCollection() : $layouts;
}
```

Even if i'm not a big fan of this solution

Reply:
The main reason was to init the field, so you can do `$theme->getLayouts()->count()` even with an empty theme (no need to check for null)